service-workers/service-worker.js

```diff
@@ -1,3 +1,56 @@
+var VERSION = '1.4.80'
+var CACHE_NAME = 'ubyssey-cache-v' + VERSION;
+var urlsToCache = [
+  '/static/js/vendors-' + VERSION + '.js',
+  '/static/js/main-' + VERSION + '.js',
+  '/static/js/article-' + VERSION + '.js',
+  '/static/images/ubyssey-logo.svg'
+];
```

Comment:
I think we should additionally cache:
- all the .woff font files. 
- main-version.css
- blockadblock-version.js
- dfp-version.js